crates/sui-core/src/epoch/randomness.rs

```diff
@@ -507,6 +519,20 @@ impl Inner {
             }
         }
 
+        // If we ran out of time, mark DKG as failed.
+        if !self.dkg_output.initialized()
```

Comment:
make this an early return pls

Reply:
realizing that even if I make it an early return, I still have to double-check the last one, since it might be initialized right above. I think partly-early-return and partly-not is more confusing, so left this as-is.